index.d.ts

```diff
@@ -545,11 +545,16 @@ declare namespace Eris {
     [key: string]: {};
   }
 
+  interface AllowedMentions {
+    everyone?: boolean;
+    roles?: boolean | string[];
+    users?: boolean | string[];
+  }
   type MessageContent = string | AdvancedMessageContent;
   type AdvancedMessageContent = {
     content?: string;
     tts?: boolean;
-    disableEveryone?: boolean;
+    allowedMentions?: AllowedMentions;
```

Comment:
I think it would be nice to continue using this option by implementing this option to disable everyone and here mentions but allow user and role mentions by default. I do understand that using allowedMentions is more granular but it makes this into a breaking change.

Reply:
To clarify, you're suggesting default behavior be kept, not keeping the disableEveryone option itself?

Reply:
if disableEveryone is true default allowedMention => 
```js
{
  parse: ["roles", "users"]
}
```